build.cmd

```diff
@@ -6,8 +6,18 @@ setlocal
 ::       means that that rebuilding cannot successfully delete the task
 ::       assembly. 
 
-echo %2 | find /I "outerloop"
-if "%errorlevel%" equ "0" (
+set outloop=false
+
+REM this is a temporary step until the build tools provide a proper hook. 
+REM it will need to deal with multiple include and exclude categories.
+REM See dotnet/corefx#1477
+
+echo %* | findstr /i /C:"/p:WithCategories=OuterLoop"  1>nul
```

Comment:
This will break for syntaxes like /p:Withcategories=Custom,OuterLoop. I really think simpler is better. The appearance of OuterLoop anywhere on the cmd line is good enough for now. It would even survive a rename of WithCategories. We should actively chat on the 1477 issue so they know our requirements.